concent_api/core/utils.py

```diff
@@ -84,3 +85,19 @@ def hex_to_bytes_convert(client_public_key: str):
     key_bytes = decode_hex(client_public_key)
     assert len(key_bytes) == GOLEM_PUBLIC_KEY_LENGTH
     return key_bytes
+
+
+def are_eth_addresses_and_singularities_equal_in_all_subtask_results_accepted_list(force_payment: ForcePayment) -> bool:
```

Comment:
You don't need to pass whole `ForcePayment` message; it will suffice if you pass just a `subtask_results_accepted_list`

Reply:
as disuccesd, please remove this function and refactor `verify_message_subtask_results_accepted` and enhance it with check of `requestor_ethereum_address`